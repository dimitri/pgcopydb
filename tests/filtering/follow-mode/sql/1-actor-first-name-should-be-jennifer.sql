-- actor first_name should be JENNIFER
SELECT
    first_name
FROM
    public.actor
WHERE
    actor_id = 1;
