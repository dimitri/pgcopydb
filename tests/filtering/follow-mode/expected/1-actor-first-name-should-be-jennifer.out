-[ RECORD 1 ]--------
first_name | JENNIFER

